### PR TITLE
Remove uses of add/sub/mul

### PR DESCRIPTION
**Pull request by area**
These are no longer needed with the Solidity upgrade, and in #1053 we agreed would be removed in a separate PR.

There's a new slither release that flagged a few new things which I've dealt with as I saw appropriate.

### CODE REVIEW

File: contracts/extensions/FundingQueue.sol [hunk 1]
```diff
@@ -268,11 +268,7 @@ contract FundingQueue is ColonyExtension, BasicMetaTransaction {
 
     // Update the user's reputation backing
     uint256 prevBacking = supporters[_id][msgSender()];
-    if (_backing >= prevBacking) {
-      proposal.totalSupport = add(proposal.totalSupport, sub(_backing, prevBacking));
-    } else {
-      proposal.totalSupport = sub(proposal.totalSupport, sub(prevBacking, _backing));
-    }
+    proposal.totalSupport = proposal.totalSupport - prevBacking + _backing;
```
Comment (kronosapiens) [inline contracts/extensions/FundingQueue.sol:271]: Parentheses

File: contracts/reputationMiningCycle/ReputationMiningCycleRespond.sol [hunk 2]
```diff
@@ -600,7 +600,7 @@ contract ReputationMiningCycleRespond is ReputationMiningCycleCommon {
   // e.g. for log entry with 6 updates, the relative update number range is [0 .. 5] (inclusive)
   function getRelativeUpdateNumber(uint256[26] memory u, ReputationLogEntry memory logEntry) internal view returns (uint256) {
     uint256 nLeaves = IColonyNetwork(colonyNetworkAddress).getReputationRootHashNLeaves();
-    uint256 updateNumber = sub(sub(disputeRounds[u[U_ROUND]][u[U_IDX]].lowerBound, 1), nLeaves);
+    uint256 updateNumber = disputeRounds[u[U_ROUND]][u[U_IDX]].lowerBound - 1 - nLeaves;
```
Comment (kronosapiens) [inline contracts/reputationMiningCycle/ReputationMiningCycleRespond.sol:603]: Parentheses

File: contracts/colony/ColonyFunding.sol [hunk 3]
```diff
@@ -74,21 +74,21 @@ contract ColonyFunding is ColonyStorage { // ignore-swc-123
     uint remainder;
     if (_token == address(0x0)) {
       // It's ether
-      toClaim = sub(sub(address(this).balance, nonRewardPotsTotal[_token]), fundingPots[0].balance[_token]);
+      toClaim = (address(this).balance - nonRewardPotsTotal[_token]) - fundingPots[0].balance[_token];
```
[the remark this answers is not in the document]
Comment (area) [reply contracts/colony/ColonyFunding.sol:77]: I don't understand what extra parentheses you're looking for in this instance?

**Comment by area** [comment]
I don't really think brackets in expressions with operators that have the same precedence add much, but I've added them in where I think I knew what you meant.

**Comment by kronosapiens** [comment]
@area my thinking was that even if not strictly necessary, they are useful in clarifying the intention behind non-associative operations.

File: contracts/colony/ColonyFunding.sol (hunk 3 above)
[the remark this answers is not in the document]
Comment (kronosapiens) [reply contracts/colony/ColonyFunding.sol:77]: ((a - b) - c), which is not equivalent to (a - (b - c)). I know the operations are processed left-to-right, but I think disambiguation helps.

**Review by kronosapiens — APPROVED** [review]

File: contracts/colony/ColonyFunding.sol (hunk 3 above)
[the remark this answers is not in the document]
Comment (area) [reply contracts/colony/ColonyFunding.sol:77]: When submitted for review, it had `(a-b) -c`, which I believe is unambiguous?

[the remark this answers is not in the document]
Comment (kronosapiens) [reply contracts/colony/ColonyFunding.sol:77]: I must have missed that

**Pull request merged**